index.html

```diff
@@ -82,90 +82,7 @@ <h3>Multi-Post Stories</h3>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
-        <div class="see-project-new">
-          <div class="project-new">
-            <h3>Data Dashboard Healthcare</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <div class="skills-new">
-              <ul>
-                <li><span>html</span></li>
-                <li><span>bootstrap</span></li>
-                <li><span>Ruby</span></li>
-              </ul>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
-        <div class="see-project-new">
-          <div class="project-new">
-            <h3>Website Protfolio</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <div class="skills-new">
-              <ul>
-                <li><span>html</span></li>
-                <li><span>bootstrap</span></li>
-                <li><span>Ruby</span></li>
-              </ul>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
-        <div class="see-project-new">
-          <div class="project-new">
-            <h3>Profesional Art Printing Data</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <div class="skills-new">
-              <ul>
-                <li><span>html</span></li>
-                <li><span>bootstrap</span></li>
-                <li><span>Ruby</span></li>
-              </ul>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
-        <div class="see-project-new">
-          <div class="project-new">
-            <h3>Data Dashboard Healthcare</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <div class="skills-new">
-              <ul>
-                <li><span>html</span></li>
-                <li><span>bootstrap</span></li>
-                <li><span>Ruby</span></li>
-              </ul>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
-        <div class="see-project-new">
-          <div class="project-new">
-            <h3>Website Protfolio</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <div class="skills-new">
-              <ul>
-                <li><span>html</span></li>
-                <li><span>bootstrap</span></li>
-                <li><span>Ruby</span></li>
-              </ul>
-            </div>
-          </div>
-          <button>See Projects</button>
-        </div>
+        
       </div>
       
     </div>
```

Comment:
- The must also be a popup for the "Multi-Post Stories" project card. Currently, nothing happens when I click on the `See project` button.

![image](https://user-images.githubusercontent.com/38088818/173781429-d548dbaf-310f-49bf-843c-bb180544fed0.png)

Otherwise, everything is perfect. Fantastic job!!! 👏🏾